### ISSUE TITLE
jQuery remove() instead of detach()

### ISSUE THREAD

**Issue by mitar** [DiscoverMeteor/book#184]
In a jQuery example, you for `Posts.find().observe`, you have:

```
$('ul li#' + post._id).detach();
```

Probably it would be better if it would be:

```
$('ul li#' + post._id).remove();
```

Because detaching is meant more [when you are planing to reinsert a DOM element back into the DOM](http://api.jquery.com/detach/).

**Comment by SachaG** [comment]
Makes sense, thanks :)
